streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorTopologyTest.java

```diff
@@ -1729,30 +1264,6 @@ public void process(final Record<String, String> record) {
         }
     }
 
-    /**
-     * A processor that stores each key-value pair in an in-memory key-value store registered with the context.
-     */
-    @SuppressWarnings("deprecation") // Old PAPI. Needs to be migrated.
-    protected static class OldAPIStatefulProcessor extends org.apache.kafka.streams.processor.AbstractProcessor<String, String> {
```

Comment:
There is already a `StatefulProcessor` for the new PAPI -- nothing to be migrated -- deleting.